documentation/troubleshooting.md

```diff
@@ -1,18 +1,18 @@
 # Troubleshooting Guide
 
-Here is our guide for diagnosing specific issues with `Dotnet-Monitor`. This is an ongoing effort to add issues and solutions to this guide and is never complete but you can help us by letting us know in the issues sections if we should add something here
+Here is our guide for diagnosing specific issues with `dotnet monitor`. This is an ongoing effort to add issues and solutions to this guide; you can help us by letting us know in the Issues section if we should add something here.
 
 ## Problem: In Azure App Service on Linux, dotnet-monitor is consuming lots of CPU time
 
-If you find that the `dotnet-monitor` is consuming lots of CPU time, this typically means that there is a high load of logging statements being emitted from the process. `dotnet-monitor` is used to capture `AppServiceAppLogs` and `AllMetrics` from your process before sending them onto other systems; these settings are configured under Monitoring > Diagnostic Settings in the Azure Portal. `dotnet-monitor` is also always enabled to collect a baseline level of metrics from the process to display in the Azure Portal. However, metrics monitoring is fast enough that it should not show up as an impact on CPU time.
+If you find that `dotnet monitor` is consuming lots of CPU time, this typically means that there is a high load of logging statements being emitted from the process. `dotnet monitor` is used to capture `AppServiceAppLogs` and `AllMetrics` from your process before sending them onto other systems; these settings are configured under Monitoring > Diagnostic Settings in the Azure Portal. `dotnet monitor` is also always enabled to collect a baseline level of metrics from the process to display in the Azure Portal. However, metrics monitoring is fast enough that it should not show up as an impact on CPU time.
```

Comment:
"fast enough" -> "lightweight such" or "low impact such"

The metrics are not fast, they are low impact.